### ISSUE TITLE
Localization for React front-end

### ISSUE THREAD

**Issue by brew** [okfn/opendatacensus#864]
As part of localization of the Survey (#844), hardcoded strings in the React jsx templates should be able to be localized. This affects the submission entry form.

**Comment by morchickit** [comment]
That's connected to #868 , no? @brew

**Comment by brew** [comment]
No. Separate issue. This relates only to localization within the entry form (which uses React). #868 refers to problems with the old codebase that have been resolved in the new version.
